docs/synthetics/ci-cd-integration.md

```diff
@@ -0,0 +1,300 @@
+title: CI/CD Integration
```

Comment:
Will "CI/CD Integration" be informative enough in SERPs?

Reply:
Not sure what is SERPs

Reply:
SERP - Search Engine Results Page. e.g. Google search results page.

Reply:
aha. This is how other vendors are also promoting this feature. Did you have something like `trigger runs via API` in mind?

Reply:
Ah, ignore my comment.  I'm thinking <title>, but this will end up being a page heading.  Ignore me.